recipes/pygsl_lite/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pygsl_lite" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygsl_lite-{{ version }}.tar.gz
+  sha256: d23944f034553a6e350305b088b6261f5854964cc481ecc15a0c5aca61560632
+
+
+build:
+  ignore_run_exports:
+    - numpy
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - gsl
+    - numpy
+    - pip
+    - python
+    - setuptools
+    - setuptools-scm >=3.4.3
+    - swig >=4.0
```

Comment:
since `swig` is a build tool, not a library, this should be in `requirements/build`, not `requirements/host`.

Reply:
Thanks, fixed!